openedx/features/pakx/lms/pakx_admin_app/templates/pakx_admin_app/edx_ace/registrationnotification/email/body.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
```

Comment:
We can move HTML files to pakx-theme repo, reference PR
https://github.com/PakistanX/pakx-theme/pull/86/files

Reply:
can we get the template the sales team already using if we are not already using it?